src/v/raft/consensus.cc

```diff
@@ -1221,6 +1221,65 @@ consensus::abort_configuration_change(model::revision_id revision) {
     co_return errc::success;
 }
 
+template<typename Func>
+ss::future<std::error_code>
+consensus::force_replace_configuration_locally(Func&& f) {
+    return _op_lock.get_units()
```

Comment:
nit: coroutines?

Reply:
iirc there is some issues with coroutine templates in at least the version of clang we are using 14. that might be what's happening here?

Reply:
> iirc there is some issues with coroutine templates in at least the version of clang we are using 14.

right, I remember experiencing those too... but if it compiles then it is okay :)